Makefile

```diff
@@ -74,13 +74,12 @@ docs: ## generate Sphinx HTML documentation, including API docs
 servedocs: docs ## compile the docs watching for changes
 	watchmedo shell-command -p '*.rst' -c '$(MAKE) -C docs html' -R -D .
 
-release: clean ## package and upload a release
-	python setup.py sdist upload
-	python setup.py bdist_wheel upload
+release: clean ## package and upload a release (requires a `~/.pypirc` file)
+	twine upload dist/*
```

Comment:
Assuming that's the correct if/then syntax for a Makefile (i.e. I'm not sure), this would give a better user experience than getting an error message from twine.